Description: Upstream changes introduced in version 2.1.10-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 slurm-llnl (2.1.10-1) unstable; urgency=low
 .
   * New upstream release
   * Setting /usr/bin/mail as default mailer (Closes: #588862)
 .
 The person named in the Author field signed this changelog entry.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Bug-Debian: http://bugs.debian.org/588862

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- slurm-llnl-2.1.10.orig/src/common/read_config.h
+++ slurm-llnl-2.1.10/src/common/read_config.h
@@ -83,7 +83,7 @@ extern char *default_plugstack;
 #define DEFAULT_KILL_ON_BAD_EXIT    0
 #define DEFAULT_KILL_TREE           0
 #define DEFAULT_KILL_WAIT           30
-#define DEFAULT_MAIL_PROG           "/usr/bin/mail"
+#define DEFAULT_MAIL_PROG           "/bin/mail"
 #define DEFAULT_MAX_JOB_COUNT       5000
 #define DEFAULT_MEM_PER_CPU         0
 #define DEFAULT_MAX_MEM_PER_CPU     0
--- slurm-llnl-2.1.10.orig/doc/man/man1/strigger.1
+++ slurm-llnl-2.1.10/doc/man/man1/strigger.1
@@ -211,7 +211,7 @@ argument to the script by SLURM).
 	strigger \-\-set \-\-node \-\-down \\
 	         \-\-program=/usr/sbin/slurm_admin_notify
 	# Notify administrator using by e\-mail
-	/usr/bin/mail slurm_admin@site.com \-s NodesDown:$*
+	/bin/mail slurm_admin@site.com \-s NodesDown:$*
 
 	> strigger \-\-set \-\-node \-\-down \\
 	           \-\-program=/usr/sbin/slurm_admin_notify
--- slurm-llnl-2.1.10.orig/doc/man/man5/slurm.conf.5
+++ slurm-llnl-2.1.10/doc/man/man5/slurm.conf.5
@@ -652,7 +652,7 @@ not explicitly listed in the job submiss
 .TP
 \fBMailProg\fR
 Fully qualified pathname to the program used to send email per user request.
-The default value on Debian systems is "/usr/bin/mail".
+The default value is "/bin/mail".
 
 .TP
 \fBMaxJobCount\fR
--- slurm-llnl-2.1.10.orig/doc/html/configurator.html.in
+++ slurm-llnl-2.1.10/doc/html/configurator.html.in
@@ -159,7 +159,7 @@ function displayfile()
    "#JobRequeue=1 <br>" +
    "#KillOnBadExit=0 <br>" +
    "#Licenses=foo*4,bar <br>" +
-   "#MailProg=/usr/bin/mail <br>" +
+   "#MailProg=/bin/mail <br>" +
    "#MaxJobCount=5000 <br>" +
    "#MaxTasksPerNode=128 <br>" +
    "MpiDefault=" + get_radio_value(document.config.mpi_default) + "<br>" +
